axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineServiceImpl.java

```diff
@@ -333,6 +341,55 @@ public BigDecimal getCutOffProrataAmount(MoveLine moveLine, LocalDate moveDate)
     return prorata.multiply(moveLine.getCurrencyAmount()).setScale(2, RoundingMode.HALF_UP);
   }
 
+  @Override
+  public void computeCutOffProrataAmount(AccountingBatch accountingBatch) throws AxelorException {
+    Company company = accountingBatch.getCompany();
+    LocalDate moveDate = accountingBatch.getMoveDate();
+    Journal researchJournal = accountingBatch.getResearchJournal();
+    int accountingCutOffTypeSelect = accountingBatch.getAccountingCutOffTypeSelect();
+
+    int offset = 0;
+    List<MoveLine> moveLineList;
+    Query<MoveLine> moveLineQuery =
+        cutOffService.getMoveLines(company, researchJournal, moveDate, accountingCutOffTypeSelect);
+
+    while (!(moveLineList = moveLineQuery.fetch(10, offset)).isEmpty()) {
+
+      for (MoveLine moveLine : moveLineList) {
+        ++offset;
+
+        this.computeCutOffProrataAmount(moveLine, moveDate);
+      }
+
+      JPA.clear();
+    }
+  }
+
+  @Override
+  @Transactional(rollbackOn = {Exception.class})
+  public MoveLine computeCutOffProrataAmount(MoveLine moveLine, LocalDate moveDate) {
+    if (moveDate != null
+        && moveLine.getCutOffStartDate() != null
+        && moveLine.getCutOffEndDate() != null) {
+      BigDecimal daysProrata =
+          BigDecimal.valueOf(ChronoUnit.DAYS.between(moveDate, moveLine.getCutOffEndDate()));
+      BigDecimal daysTotal =
+          BigDecimal.valueOf(
+              ChronoUnit.DAYS.between(moveLine.getCutOffStartDate(), moveLine.getCutOffEndDate()));
```

Comment:
Please add one day here, like it's done in this PR: https://github.com/axelor/axelor-open-suite/pull/9700/files#diff-8efe19f38aa0f9eff3d1d22e09223ab3999367466daecfeca24acc9312c8f550R327